src/main/java/Duke.java

```diff
@@ -1,10 +1,272 @@
+                    "   .     .        jo  o    |     .            +\n" +
+                    "                 ( ~T~     j                     +     .\n" +
+                    "      +           >._-' _./         +\n" +
+                    "               /| ;-\"~ _  l\n" +
+                    "  .           / l/ ,-\"~    \\     +\n" +
+                    "              \\//\\/      .- \\\n" +
+                    "       +       Y        /    Y\n" +
+                    "               l       I     !\n" +
+                    "               ]\\      _\\    /\"\\\n" +
+                    "              (\" ~----( ~   Y.  )\n" +
+                    "          ~~~~~~~~~~~~~~~~~~~~~~~~~~\n";
+    public static final String WARREN_ASCII =
+            ",--.   ,--.\n" +
+                    "|  |   |  | ,--,--.,--.--.,--.--. ,---. ,--,--,\n" +
+                    "|  |.'.|  |' ,-.  ||  .--'|  .--'| .-. :|      \\\n" +
+                    "|   ,'.   |\\ '-'  ||  |   |  |   \\   --.|  ||  |\n" +
+                    "'--'   '--' `--`--'`--'   `--'    `----'`--''--'\n";
+    public static final String GREETING_MESSAGE =
+            "Hello,\n" + WARREN_ASCII +
+                    "What can I do for you? (enter 'hello' for available commands!)";
+    public static final String FAREWELL_MESSAGE = "Bye. Hope to see you again soon!";
+
+    /**
+     * Simply prints out a dividing line to standard output
+     */
+    public static void printDivider() {
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * Prints out the greeting message to standard output
+     */
+    public static void printGreet() {
+        printDivider();
+        System.out.println(RABBIT_ASCII);
+        printDivider();
+        System.out.println(GREETING_MESSAGE);
+        printDivider();
+    }
+
+    /**
+     * Prints out the farewell message to standard output
+     */
+    public static void printFarewell() {
+        System.out.println(FAREWELL_MESSAGE);
+        printDivider();
+    }
+
+    /**
+     * Takes the provided input string and saves it into the array of Task "tasks"
+     * If the number of tasks exceed MAX_COUNT, the input string will not be saved.
+     *
+     * @param input    Input string to be saved
+     * @param taskType Type of task to save as
+     */
+    public static void saveInput(String input, Task.TaskType taskType) {
+        if (inputCount >= MAX_COUNT) {
+            System.out.println(ERROR_INPUT_LIMIT_REACHED +
+                    "The last entry, \"" + input + "\", will not be added to the list.");
+        } else {
+            String[] inputs;
+            boolean isTaskNameOnly;
+            switch (taskType) {
+            case TODO:
+                tasks[inputCount] = new Todo(input);
+                break;
+            case EVENT:
+                inputs = input.split(REGEX_PATTERN_AT, 2);
+                isTaskNameOnly = inputs.length < 2;
```

Comment:
Maybe this could be put in a method to increase readability?

Reply:
Agree with @laiisaac. Extracting out the code under each switch statement into different method will allow scalability and easier debugging